### ISSUE TITLE
Controls on Website too high

### ISSUE THREAD

**Issue by EdoVro** [44670/44vba#19]
I’m on a 2nd generation iPhone SE, and been using the web version of the emulator for some time through the Home Screen. It runs well, but in vertical aspect ratio, the controls don’t appear where they’re meant to be. Image is attached.
![37BE9523-F769-44D0-A318-4868D8C6C8A1](https://user-images.githubusercontent.com/91586132/217080762-6be086fb-f413-424a-9f07-f89c755b1acb.png)

**Comment by 44670** [comment]
Should be a bug, try opening it again?

**Comment by EdoVro** [comment]
No, they’re always at the top every time I open the emulator. The DS emulator doesn’t do this, just the GBA one.

**Comment by 44670** [comment]
I see, will investigate into it.

**Comment by guaibaolu** [comment]
> No, they’re always at the top every time I open the emulator. The DS emulator doesn’t do this, just the GBA one.

Hi, could you please tell me how to run DS emulator? a website like https://gba.44670.org/?? thanks
